src/Traits/TransactionTrait.php

```diff
@@ -136,17 +159,30 @@ trait TransactionTrait
     /**
      * Payment source.
      *
-     * @Groups({"create", "update"})
+     * @Groups({"get"})
      *
      * @Assert\Type(type="\EoneoPay\PhpSdk\Endpoints\PaymentSource")
      *
      * @var \EoneoPay\PhpSdk\Endpoints\PaymentSource|null
      */
     protected $paymentSource;
 
+    /**
+     * Recurring ID.
+     *
+     * @Groups({"get"})
+     *
+     * @Assert\Type(type="string")
+     *
+     * @var string|null
+     */
+    protected $recurring_id;
```

Comment:
camel case?

Reply:
I'm not actually sure what recurring id is, it feels like it might have been a feature that got moved into subscriptions

Reply:
This guy is still in here?

Reply:
He's like that annoying last drunk guy at a party, just won't leave.

Reply:
Yeah, it looks like you can set this on creation.

Not sure we should have it, but from following the trail of PrimaryController in payments, this can be set. If it shouldn't be able to set, we should mark it as deprecated, with a bug ticket.

Reply:
Fixed. Ticket to fix created https://eonx.atlassian.net/browse/PYMT-1552